### ISSUE TITLE
Prompt to rate and review app

### ISSUE THREAD

**Issue by AW-STJ** [AlphaWallet/alpha-wallet-ios#2793]
Linking to the issue in Android repo

https://github.com/AlphaWallet/alpha-wallet-android/issues/1856

**Comment by hboon** [comment]
We already use the `SKStoreReviewController` API provided by iOS for this. It is the officially recommended way. Too many dark patterns have popped up over the years before this existed. It lets apps request to iOS that the user be shown a prompt to rate the app. The prompt has a few characteristics:

* is out of process, the app can't control if it is actually shown and don't know if it is shown or what is the user's choice
* no matter how many times the app tries, the prompt only appears a maximum number of times within a period of time (as of writing, 3 times within 12 months)

We currently call this API the 6th time the current build has been launched (the number is arbitrary; we inherited it).

I know I suggested that we prompt the user to rate after they complete a transaction successfully. I'll add that too #2794. But we can't remove this somewhat arbitrary call above because the actual times that the prompt appear is beyond our control — removing it might cause user to never be prompted unless they are long term users.

So no other work needed for iOS unless we want to go for a elaborate approach.
